nilearn/input_data/multi_nifti_masker.py

```diff
@@ -42,6 +42,16 @@ class MultiNiftiMasker(NiftiMasker, CacheMixin):
     standardize: boolean, optional
         If standardize is True, the time-series are centered and normed:
         their mean is put to 0 and their variance to 1 in the time dimension.
+        The ``standardize`` parameter is deprecated and will be removed, use
+        ``normalize="std"`` to standardize.
+
+    normalize: {"psc", "std", None}, optional
+        The time-series normalization method.
+        If 'psc' (percent signal change), input time-series means in the time
+        dimension are put to 100 prior to any temporal preprocessing.
+        If 'std', the output time-series are centered and normed:
```

Comment:
Here, I would say "... are standardized, ie centered and normed". The goal being that people who search for "standardize" in the docs find this argument.